src/couch/src/couch_debug.erl

```diff
@@ -891,6 +904,23 @@ print_table(Rows, TableSpec) ->
     ),
     ok.
 
+print_report(Report) ->
+    TableSpec = [
+        {50, centre, info},
+        {100, right, value}
+    ],
+    io:format("~s~n", [format(TableSpec)]),
+    lists:map(
+        fun({InfoKey, Value}) ->
+            TableSpec1 = [
```

Comment:
If you would follow the suggestion about providing `print_report/2` version of the function you wouldn't be able to have a second copy of a TableSpec. One option is to derive second TableSpec from the first one. I though about something like.

```
TableSpec1 = lists:keymap(fun
   (value) -> InfoKey;
   (X) -> X
end, 3, TableSpec).
```

However there is a better solution (in terms of performance). This function assumes (as it is currently written in your PR) that the table has only 2 columns. Therefore we can pattern match on the shape:

```
print_report(Report, [Column1, {W2, A2, _}] = TableSpec) ->
    io:format("~s~n", [format(TableSpec)]),
    lists:map(
        fun({InfoKey, Value}) ->
             TableSpec1 = [Column1, {W2, A2, InfoKey}],
            io:format("~s~n", [table_row(InfoKey, 2, [{InfoKey, Value}], TableSpec1)])
        end,
        Report
    );
print_report(_Report, TableSpec) when is_list(TableSpec) ->
    throw({incorrect_spec, lists:flatten(io_lib:format("We expect a table with 2 rows, but got ~p", [length(TableSpec)]))});
print_report(_Report, TableSpec) ->
    throw({incorrect_spec, "TableSpec must be a list, i.e. [{20, centre, info},{100, right, value}]"}}).
```